lib/trino-parquet/src/main/java/io/trino/parquet/BloomFilterStore.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.parquet;
+
+import com.google.common.annotations.VisibleForTesting;
+import io.airlift.log.Logger;
+import org.apache.parquet.column.values.bloomfilter.BlockSplitBloomFilter;
+import org.apache.parquet.column.values.bloomfilter.BloomFilter;
+import org.apache.parquet.format.BloomFilterHeader;
+import org.apache.parquet.format.Util;
+import org.apache.parquet.hadoop.metadata.ColumnChunkMetaData;
+import org.apache.parquet.hadoop.metadata.ColumnPath;
+import org.apache.parquet.io.SeekableInputStream;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+public class BloomFilterStore
+{
+    private static final Logger LOG = Logger.get(BloomFilterStore.class);
+
+    private final Map<ColumnPath, BloomFilter> cache = new HashMap<>();
+    private final ParquetDataSource dataSource;
+
+    @VisibleForTesting
+    BloomFilterStore()
+    {
+        this.dataSource = null;
+    }
```

Comment:
This should not be required to test this class. You can do something similar to `TestHdfsParquetDataSource#testPlanReadOrdering` if it's really needed for testing

Reply:
I created dummy class and used inheritance instead.